docs/user_guide/getting_started_with_moonstone.rst

```diff
@@ -141,5 +141,21 @@ Normalization
 Analysis
 """"""""
 
-Visualization
-"""""""""""""
+Plot
+"""""
+
+At any point of your work on your data, Moonstone provides standard plots to visualize your data and metadata.
+
+Plot available for data :
```

Comment:
For the list, we can update this later, when the other PR is merged (we might have conflict issue here by the way)